dagshub/upload/wrapper.py

```diff
@@ -24,6 +26,66 @@ def get_default_branch(src_url, owner, reponame, auth):
     return res.json().get('default_branch')
 
 
+def create_repo(repo_name, description="", private=False, auto_init=False, gitignores="", license="", readme="", template="notebook-template"):
+    import logging
+    logging.basicConfig(level=logging.DEBUG)
+    import dagshub.auth
+    from dagshub.auth.token_auth import HTTPBearerAuth
+
+    username = config.username
+    password = config.password
+    if username is not None and password is not None:
+        return username, password
+    try:
+        token = dagshub.auth.get_token(code_input_timeout=0)
+    except dagshub.auth.OauthNonInteractiveShellException:
+        logger.debug("Failed to perform OAuth in a non interactive shell")
+    if token is not None:
+        auth = HTTPBearerAuth(token)
+
+    userRes = requests.get(urllib.parse.urljoin(config.host, USER_INFO_URL), auth=auth)
+    userJson = userRes.json()
+    if username is None:
+        username = userJson["username"]
+
+    repoRes = requests.get(urllib.parse.urljoin(config.host, REPO_INFO_URL.format(
+        owner=username,
+        reponame=repo_name
+    )))
+    if repoRes.status_code == HTTPStatus.OK:
+        return Repo(owner=username, name=repo_name, username=username, token=token)
+
+    if license is None and readme is None and template is None and gitignores is None:
+        raise RuntimeError(
+            "In order to create the repository please choose a project template. "
+            "Our options are cookiecutter-dagshub-dvc or notebook-template.")
+
+    data = {
+        "name": repo_name,
+        "description": description,
+        "private": private,
+        "auto_init": auto_init,
+        "gitignores": gitignores,
+        "license": license,
+        "readme": readme,
+        "project_template": template,
+    }
+
+    res = requests.post(
+        urllib.parse.urljoin(config.host, REPO_CREATE_URL),
```

Comment:
Add an optional parameter for the repo to be created under an org with the url pattern: `POST /org/:org/repos`